### PR TITLE
Fix CSS & Check login issues

### PR DESCRIPTION
**Pull request by jp-13**
Related to #196

### CODE REVIEW

File: safaribooks.py [hunk 1]
```diff
@@ -377,14 +377,17 @@ def __init__(self, args):
             cover_html = self.parse_html(
                 html.fromstring("<div id=\"sbo-rt-content\"><img src=\"Images/{0}\"></div>".format(self.cover)), True
             )
+            cover_css = self.parse_css(
+                html.fromstring("<div id=\"sbo-rt-content\"><img src=\"Images/{0}\"></div>".format(self.cover))
+            )
```
Comment (antoineco) [inline safaribooks.py:382]: You have a bunch of trailing whitespaces on that line.

**Comment by antoineco** [comment]
I tried your changes. My `cookies.json` file gets wiped every time I run the script now.

**Comment by antoineco** [comment]
Thanks for the changes!

Are you also experiencing the following error on your side?

```console
$  python3 safaribooks.py 9781492057888
...
[*] Output directory:
    /home/acotten/git/safaribooks/Books/Building Event-Driven Microservices (9781492057888)
[-] Downloading book contents... (9 chapters)
    [############################################################################################################] 100%[-] Downloading book CSSs... (4 files)
[#] ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[#] ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[#] Error trying to retrieve this CSS: /home/acotten/git/safaribooks/Books/Building Event-Driven Microservices (9781492057888)/OEBPS/Styles/Style02.css
    From: https://learning.oreilly.com/static/css/annotator.e3b0c44298fc.css
[#] Error trying to retrieve this CSS: /home/acotten/git/safaribooks/Books/Building Event-Driven Microservices (9781492057888)/OEBPS/Styles/Style01.css
    From: https://learning.oreilly.com/static/CACHE/css/output.8054605313ed.css
Process Process-2:
Process Process-3:
Traceback (most recent call last):
  File "/usr/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "safaribooks.py", line 854, in _thread_download_css
    s.write(response.content)
AttributeError: 'int' object has no attribute 'content'
Traceback (most recent call last):
  File "/usr/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "safaribooks.py", line 854, in _thread_download_css
    s.write(response.content)
AttributeError: 'int' object has no attribute 'content'
[-] Downloading book images... (21 files)
    [############################################################################################################] 100%[-] Creating EPUB file...
[*] Done: /home/acotten/git/safaribooks/Books/Building Event-Driven Microservices (9781492057888)/9781492057888.epub

    If you like it, please * this project on GitHub to make it known:
        https://github.com/lorenzodifuccia/safaribooks
    e don't forget to renew your Safari Books Online subscription:
        https://learning.oreilly.com

[!] Bye!!
```

File: safaribooks.py (hunk 1 above)
Comment (jp-13) [reply safaribooks.py]: Fixed, thanks!

**Comment by noxymon** [comment]
this pull request, seems working with CSS not found issue, 
although the multiprocessing method seems exceeding O'Reilly the limit.

**Pull request closed without merging**